app/src/main/java/com/android/bookswap/ui/map/Map.kt

```diff
@@ -89,13 +92,20 @@ fun MapScreen(
   val cameraPositionState = rememberCameraPositionState()
   // Get the user's current location
   val latitude = geolocation.latitude.collectAsState()
+  val context = LocalContext.current
+  var isOnline = remember { isNetworkAvailable(context) }
   val longitude = geolocation.longitude.collectAsState()
   // Start location and books updates
   LaunchedEffect(Unit) {
-    bookManagerViewModel.startUpdatingBooks()
+    if (isOnline) {
+      bookManagerViewModel.startUpdatingBooks()
+    } else {
+      Toast.makeText(context, "Please connect to Internet to actualise", Toast.LENGTH_SHORT).show()
```

Comment:
I think the word you are looking for here is update or even show, if possible please change the word "actualise", as in english that means to make something come true, to make it into reality, which is way off from what is here looked for.